backend/de.metas.manufacturing/src/main/sql/postgresql/system/30-de.metas.adempiere.libero/5610460_sys_gh11905_pp_product_planning_pp_product_bom.sql

```diff
@@ -0,0 +1,35 @@
+drop index ppproductbomunique;
+
+create unique index ppproductbomunique
+	on pp_product_bom (ad_client_id, value, validfrom);
+
+insert into pp_product_bomversions(PP_Product_BOMVersions_ID,
+                                   AD_Client_ID,
+                                   AD_Org_ID,
+                                   Created,
+                                   Createdby,
+                                   IsActive,
+                                   Updated,
+                                   UpdatedBy,
+                                   M_Product_ID,
+                                   Name)
+select distinct nextval('pp_product_bomversions_seq'),
+                max(AD_Client_ID),
+                max(AD_Org_ID),
+                getdate(),
```

Comment:
please use a hardcoded date like `'2021-11-04 15:01'`. that way we may e.g. find the migration script from the created timestamp